### PR TITLE
DNM: Focus test in CI

### PR DESCRIPTION
**Pull request by lyarwood**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/instancetype_test.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ import (
 	"kubevirt.io/kubevirt/tests/util"
 )
 
-var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-compute] Instancetype and Preferences", decorators.SigCompute, func() {
+var _ = FDescribe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-compute] Instancetype and Preferences", decorators.SigCompute, func() {
```
Comment (akalenyu) [inline tests/instancetype_test.go:44]: Would this actually work? I think ginkgo cli parameters like skip will mess this up

Comment (akalenyu) [reply tests/instancetype_test.go:44]: Last time I needed something like this I used this (note this is CDI):
https://github.com/kubevirt/containerized-data-importer/pull/2384

Comment (lyarwood) [reply tests/instancetype_test.go:44]: This came up in https://github.com/kubevirt/kubevirt/pull/10136#issuecomment-1643194689 and I wanted to quickly test it in CI

Comment (akalenyu) [reply tests/instancetype_test.go:44]: Yep so this will probably get ignored unless you remove some ginkgo params

Comment (lyarwood) [reply tests/instancetype_test.go:44]: Yup it's passing the following `--label-filter` in that ignore focused containers, cool. 

```
FUNC_TEST_LABEL_FILTER='--label-filter=(sig-compute && !(GPU,VGPU,sig-compute-migrations,exclude-native-ssh))&&(!verify-non-root)&&(!CustomSELinux)'
make functest
```

**Pull request closed without merging**